src/Simulator/IncrementalRedstoneSimulator.cpp

```diff
@@ -1160,17 +1160,33 @@ void cIncrementalRedstoneSimulator::HandlePressurePlate(int a_RelBlockX, int a_R
 		{
 			// MCS feature - stone pressure plates can only be triggered by players :D
 			cPlayer * a_Player = this->m_World.FindClosestPlayer(Vector3f(BlockX + 0.5f, static_cast<float>(a_RelBlockY), BlockZ + 0.5f), 0.5f, false);
+			static Int64 m_PlayerLastOnPlate = 0;
```

Comment:
I understand that static variables persist, but since this model of redstone simulator reuses the handler functions for every pressure plate, I believe that with multiple plates, these variables would be essentially be repeatedly overwritten?